docs/docs/schema.md

```diff
@@ -0,0 +1,367 @@
+---
+
+title: Schema
+description: Define the schema of your database.
+
+---
+
+## Overview
+
+In Drift, your schema is represented as a table. Each column in the table represents a field in your data class.
+
+Take this example of a table that stores superheroes:
+
+| ID  | Name      | Secret Name  | Age    | Height |
+| --- | --------- | ------------ | ------ | ------ |
+| 1   | Superman  | Clark Kent   | 35     | 6'2"   |
+| 2   | Batman    | Bruce Wayne  | 40     | 6'0"   |
+| 3   | Spiderman | Peter Parker | 25     | 5'10"  |
+| 4   | Ironman   | Tony Stark   | `null` | 6'1"   |
+| 5   | Thor      | `null`       | 1500   | `null` |
+
+
+Here we have a table with 5 columns: `ID`, `Name`, `Secret Name`, `Age`, and `Height`. Each row in the table represents a single superhero.
+
+Also, we need to limit how the data is stored, for instance:
+
+- `ID` and `Name` should be unique for each row.
+- `Secret Name`, `Age`, and `Height` are optional fields.
+
+You can easily define a table in Drift using the following syntax:
+
+{{ load_snippet('superhero_schema','lib/snippets/schema.dart.excerpt.json') }}
+
+After defining the schema, add the table to the database. 
+
+{{ load_snippet('superhero_database','lib/snippets/schema.dart.excerpt.json') }}
+
+
+Now, run the code generator:
+```bash
+dart run build_runner build
+```
+
+<h4>Congratulations! 🎉🎉</h4>
```

Comment:
This reads like part of a guided setup tour, but I'm not sure if this page is supposed to take that role or if it should explain tables in more detail as a reference page?

I guess it will be easier to move content around once we have the basics in place, but I'd prefer to keep the distinction between guides showing everything step by step and these in-depth reference pages that (I suppose) will mostly be used to quickly look something up.

Reply:
I'll rewrite the later parts of the page to be more "reference-like", but an friendly overview gives users a sense of security